components/Common/CrossLink/index.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import CrossLink from './index';
+
+type Story = StoryObj<typeof CrossLink>;
+type Meta = MetaObj<typeof CrossLink>;
+
+export const Prev: Story = {
+  render: () => (
+    <div className="w-[305px] bg-white p-2">
+      <CrossLink
+        direction="left"
+        text="How to install Node.js"
+        url="https://nodejs.dev/en/learn/how-to-install-nodejs/"
+      />
+    </div>
+  ),
+};
+
+export const Next: Story = {
+  render: () => (
+    <div className="w-[305px] bg-white p-2">
+      <CrossLink
+        direction="right"
+        text="How much JavaScript do you need to know to use Node.js?"
+        url="https://nodejs.dev/en/learn/how-much-javascript-do-you-need-to-know-to-use-nodejs/"
+      />
+    </div>
+  ),
+};
+
+export const PrevDark: Story = {
+  render: () => (
+    <div data-theme="dark" className="w-[305px] bg-black p-2">
```

Comment:
I am proposing to add [`@storybook/addon-themes`](https://github.com/storybookjs/storybook/tree/next/code/addons/themes), see [here](https://github.com/storybookjs/storybook/blob/next/code/addons/themes/docs/getting-started/tailwind.md) for the steps. I think it's better standardised.

<details>
<summary>Changes</summary>

```diff
diff --git a/.storybook/main.ts b/.storybook/main.ts
index 862f09d2..5699db10 100644
--- a/.storybook/main.ts
+++ b/.storybook/main.ts
@@ -3,7 +3,7 @@ import type { StorybookConfig } from '@storybook/nextjs';
 
 const config: StorybookConfig = {
   stories: ['../components/**/*.stories.tsx'],
-  addons: ['@storybook/addon-controls', '@storybook/addon-interactions'],
+  addons: ['@storybook/addon-controls', '@storybook/addon-interactions', '@storybook/addon-themes'],
   framework: { name: '@storybook/nextjs', options: {} },
   features: { storyStoreV7: true },
   docs: { autodocs: 'tag' },
diff --git a/.storybook/preview.tsx b/.storybook/preview.tsx
index 45c08cca..7a0364a3 100644
--- a/.storybook/preview.tsx
+++ b/.storybook/preview.tsx
@@ -3,7 +3,8 @@ import { Open_Sans } from 'next/font/google';
 import { SiteProvider } from '../providers/siteProvider';
 import { ThemeProvider } from '../providers/themeProvider';
 import { LocaleProvider } from '../providers/localeProvider';
-import type { Preview } from '@storybook/react';
+import { withThemeByDataAttribute } from '@storybook/addon-themes';
+import type { Preview, ReactRenderer } from '@storybook/react';
 
 import '../styles/new/index.scss';
 
@@ -49,6 +50,14 @@ export const decorators = [
       </LocaleProvider>
     </SiteProvider>
   ),
+  withThemeByDataAttribute<ReactRenderer>({
+    themes: {
+      light: '',
+      dark: 'dark',
+    },
+    defaultTheme: 'light',
+    attributeName: 'data-theme',
+  }),
 ];
 
 Object.defineProperty(NextImage, 'default', {
diff --git a/components/Common/CrossLink/index.stories.tsx b/components/Common/CrossLink/index.stories.tsx
index fb67cba3..36d4ec96 100644
--- a/components/Common/CrossLink/index.stories.tsx
+++ b/components/Common/CrossLink/index.stories.tsx
@@ -6,7 +6,7 @@ type Meta = MetaObj<typeof CrossLink>;
 
 export const Prev: Story = {
   render: () => (
-    <div className="w-[305px] bg-white p-2">
+    <div className="w-[305px]">
       <CrossLink
         direction="left"
         text="How to install Node.js"
@@ -29,8 +29,13 @@ export const Next: Story = {
 };
 
 export const PrevDark: Story = {
+  parameters: {
+    themes: {
+      themeOverride: 'dark',
+    },
+  },
   render: () => (
-    <div data-theme="dark" className="w-[305px] bg-black p-2">
+    <div className="w-[305px]">
       <CrossLink
         direction="left"
         text="How to install Node.js"
diff --git a/package-lock.json b/package-lock.json
index 92b82773..06296b96 100644
...
diff --git a/package.json b/package.json
index cde35360..f5f4803b 100644
--- a/package.json
+++ b/package.json
@@ -83,6 +83,7 @@
   "devDependencies": {
     "@storybook/addon-controls": "~7.4.0",
     "@storybook/addon-interactions": "~7.4.0",
+    "@storybook/addon-themes": "^7.4.1",
     "@storybook/nextjs": "~7.4.0",
     "@testing-library/jest-dom": "~6.1.2",
     "@testing-library/react": "~14.0.0",
diff --git a/tailwind.config.ts b/tailwind.config.ts
index 40a5366f..207fd343 100644
--- a/tailwind.config.ts
+++ b/tailwind.config.ts
@@ -111,6 +111,6 @@ export default {
     fontFamily: {
       'open-sans': ['var(--font-open-sans)'],
     },
-    darkMode: ['class', '[data-theme="dark"]'],
   },
+  darkMode: ['class', '[data-theme="dark"]'],
 } satisfies Config;
```

</details>


<img width="329" alt="Screenshot 2023-09-11 at 9 45 58 PM" src="https://github.com/nodejs/nodejs.org/assets/128208841/c2bd9a66-440f-4a3a-ba3c-f4b8e2812ce1">
<img width="328" alt="Screenshot 2023-09-11 at 9 46 18 PM" src="https://github.com/nodejs/nodejs.org/assets/128208841/4fd6909b-fa4b-4703-881d-b4800b3ea72e">

We can override the theme per story as well.
<img width="328" alt="Screenshot 2023-09-11 at 9 46 33 PM" src="https://github.com/nodejs/nodejs.org/assets/128208841/876a5de5-b196-4290-9621-c0830fb83d77">

Reply:
I mean we will need to look forward. Because if we add theme-switcher as storybook plugins we need to mind how theme-switcher (component) will interact with the plugins.

Reply:
I think you'te misunderstanding how the theme switcher plugin works.

It simply simulates the same behaviour of our regular dark/light theme. Hence why on the code snippet @HinataKah0 customised the plugin config.